bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -123,12 +129,18 @@
     private static final Pattern LABEL_PATTERN = Pattern.compile(".*?\\[.*? (.*?)\\]");
     private static final int MAX_BUTTONS = 4;
 
+    private static final String SERVLET_SORTING = "LABEL";
+
+    private final Logger logger = LoggerFactory.getLogger(ItemUIRegistryImpl.class);
+
     protected final Set<ItemUIProvider> itemUIProviders = new HashSet<>();
 
     private final ItemRegistry itemRegistry;
 
     private final Map<Widget, Widget> defaultWidgets = Collections.synchronizedMap(new WeakHashMap<>());
 
+    private String groupMembersSorting = SERVLET_SORTING;
```

Comment:
With NONE being the default now, wouldn't it be safer to also initialize this field with NONE instead of LABEL?

Reply:
Yes of course.

Reply:
Waiting for your commit then ;-)

Reply:
Done